package.json

```diff
@@ -53,7 +53,7 @@
     "jest-junit-reporter": "^1.1.0",
     "jest-pnp-resolver": "^1.0.2",
     "node-sass": "^4.11.0",
-    "office-ui-fabric-react": ">=6.0.0 <7.0.0",
+    "office-ui-fabric-react": "^6.211.3",
```

Comment:
Is this version really required? I.e. it wouldn't work with any other 6.x.x version? If it's the case then for the library it still better to specify a range from `6.211.3` to `7.0.0`. Otherwise with the specified expression even version `6.212.0` wouldn't suffice.

Reply:
thanks! 6.211.3 is the minimum version required. Changed it accordingly!.